modules/upgrade.adoc

```diff
@@ -31,7 +31,7 @@ Automatic upgrade policies are optional and if they are not set, the upgrade pol
 [id="upgrade-manual_upgrades_{context}"]
 == Manual upgrades
 
-If you opt for manual upgrades, you are responsible for updating your cluster. However, if your cluster version falls too far behind, it will be automatically updated. For more information on OpenShift life cycle policies, see link:https://access.redhat.com/support/policy/updates/openshift[version support].
+If you opt for manual upgrades, you are responsible for updating your cluster. If your cluster version falls too far behind, it will transition to a limited support status. For more information on OpenShift life cycle policies, see xref:../osd_policy/osd-life-cycle.adoc#osd-life-cycle[{product-title} update life cycle].
```

Comment:
```suggestion
If you opt for manual upgrades, you are responsible for updating your cluster. If your cluster version falls too far behind, it will transition to a limited support status. For more information on OpenShift life cycle policies, see xref:../osd_policy/osd-life-cycle.adoc[{product-title} update life cycle].
```
The anchor tag doesn't actually exist

Reply:
Oh, I hadn't noticed before that this is an xref in a module file, which we presently aren't allowed to do based on technical publishing constraints. 

I've revised this so that it adheres to the OpenShift docs guidelines of not including xrefs in modules. Also, the anchor is a requirement of our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references): 

> All links to internal content is created using xref and must have an anchor ID.

I've added an "Additional resources" section in the assembly with the xref, which is unfortunately our current workaround. I've also left a comment to update to an xref once we are able to do so.

Reply:
Here's the link to this part of the preview to see what I mean: http://file.rdu.redhat.com/bfuru/0728/OSD/upgrading/osd-upgrades.html#upgrade-manual_upgrades_osd-upgrades (you will see the Additional resources xref shows up just before the next assembly ("Manually upgrading your cluster through OCM") 😞